app/src/main/java/org/mozilla/fenix/settings/SettingsFragment.kt

```diff
@@ -269,6 +269,22 @@ class SettingsFragment : PreferenceFragmentCompat() {
             resources.getString(R.string.pref_key_delete_browsing_data_on_quit_preference) -> {
                 SettingsFragmentDirections.actionSettingsFragmentToDeleteBrowsingDataOnQuitFragment()
             }
+            resources.getString(R.string.pref_key_notifications) -> {
+                if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+                    val intent = Intent(Settings.ACTION_APP_NOTIFICATION_SETTINGS)
+                    startActivity(intent)
+                } else {
+                    (activity as HomeActivity).openToBrowserAndLoad(
```

Comment:
Do we still need the if/else if the notification settings are only shown for >=Oreo devices?